src/items/modules.md

```diff
@@ -65,27 +65,22 @@ not have a `path` attribute, the path to the file mirrors the logical [module
 path].
 
 r[items.mod.outlined.search]
-Ancestor module path components are directories, and the module's
-contents are in a file with the name of the module plus the `.rs` extension.
+Ancestor module path components are directories, and the module's contents are in a file with the
+name of the module plus the `.rs` extension. Alternatively, the module's contents can also be in a
+file called `mod.rs` in a directory with the name of the module. It is not allowed to have both
+`$name.rs` and `$name/mod.rs`.
+
 For example, the following module structure can have this corresponding
 filesystem structure:
 
 Module Path               | Filesystem Path  | File Contents
 ------------------------- | ---------------  | -------------
 `crate`                   | `lib.rs`         | `mod util;`
-`crate::util`             | `util.rs`        | `mod config;`
+`crate::util`             | `util.rs` *or* `util/mod.rs` | `mod config;`
 `crate::util::config`     | `util/config.rs` |
 
-r[items.mod.outlined.search-mod]
-Module filenames may also be the name of the module as a directory with the
-contents in a file named `mod.rs` within that directory. The above example can
-alternately be expressed with `crate::util`'s contents in a file named
-`util/mod.rs`. It is not allowed to have both `util.rs` and `util/mod.rs`.
-
 > **Note**: Prior to `rustc` 1.30, using `mod.rs` files was the way to load
-> a module with nested children. It is encouraged to use the new naming
-> convention as it is more consistent, and avoids having many files named
-> `mod.rs` within a project.
+> a module with nested children.
```

Comment:
```suggestion
> a module with nested children. Rust 2018 added the new naming convention to be more consistent with modules that don't have submodules, and to avoid having many files named `mod.rs`.
```

This keeps the description of why the new convention was added, but frames it as "Rust 2018 added" rather than "It is encouraged".

Reply:
Giving rationale for the new style without giving rationale for the old style would still constitute an implicit endorsement.

So, I prefer your wording over the status quo, but I don't think it is as impartial as the reference should be.

Reply:
I don't think the reference is required to be impartial between two ways of writing things.

In the future, when we have better alternatives for every use of `as`, I would want the reference to position `as` as the *old* way of doing things and to editorialize on the new ways being better.

Reply:
What about using a `Note`? That way the reference stays descriptive, but still documents rationale or recommendation. Those notes are "useful but out of scope" information (as defined by the convention), thus giving the opportunity to be prescriptive for those who are interested.

Reply:
If this proposed edit is the only way to get the PR in, I'm willing to accept that. I don't plan to spend a lot of energy pushing on this.